Resources/Prototypes/DeltaV/Loadouts/loadout_groups.yml

```diff
@@ -220,14 +220,21 @@
   - LawyerJumpskirtRed
   - LawyerJumpsuitGood
   - LawyerJumpskirtGood
-  
```

Comment:
pro untouch

Reply:
its bad space anyway no reason to make a separate pr to remove 2 spaces :trollface: